structuredheadings/plugin.js

```diff
@@ -219,6 +219,24 @@
 
       CKEDITOR.dialog.add("selectStyle", this.path + "dialogs/selectstyle.js");
 
+      // Indent and outdent with TAB/SHIFT+TAB key
+      editor.on("key", function (evt) {
+        if (editor.mode !== "wysiwyg") {
+          return;
+        }
+
+        var tabKey = 9;
```

Comment:
technically you can move this constant out so it's not being declared on each event invocation.  the benefit is likely minor